lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1048,30 +1062,30 @@ def test_1d_slice_coord_given(self):
         res = self.cube.slices_over(self.cube.coord("model_level_number"))
         for i, res_cube in zip(self.exp_iter_1d, res):
             expected = self.cube[:, i]
-            self.assertEqual(res_cube, expected)
+            assert res_cube == expected
 
     def test_1d_slice_nonexistent_coord_given(self):
-        with self.assertRaises(CoordinateNotFoundError):
+        with pytest.raises(CoordinateNotFoundError):
             _ = self.cube.slices_over(self.cube.coord("wibble"))
 
     def test_1d_slice_coord_name_given(self):
         res = self.cube.slices_over("model_level_number")
         for i, res_cube in zip(self.exp_iter_1d, res):
             expected = self.cube[:, i]
-            self.assertEqual(res_cube, expected)
+            assert res_cube == expected
 
     def test_1d_slice_nonexistent_coord_name_given(self):
-        with self.assertRaises(CoordinateNotFoundError):
+        with pytest.raises(CoordinateNotFoundError):
             _ = self.cube.slices_over("wibble")
 
     def test_1d_slice_dimension_given(self):
         res = self.cube.slices_over(1)
         for i, res_cube in zip(self.exp_iter_1d, res):
             expected = self.cube[:, i]
-            self.assertEqual(res_cube, expected)
+            assert res_cube == expected
 
     def test_1d_slice_nonexistent_dimension_given(self):
-        with self.assertRaisesRegex(ValueError, "iterator over a dimension"):
+        with pytest.raisesRegex(ValueError, "iterator over a dimension"):
```

Comment:
It's still `pytest.raises()`. I _believe_ the `match` argument will work with regex if provided, and also with a partial text match as seen here. Just check your arguments basically, but it's probably fine!

Reply:
Yes, you're right! Just missed this one, thanks for the catch!